redwood-layout-schema/docs_images/box1.png

```diff
@@ -0,0 +1,3 @@
+version https://git-lfs.github.com/spec/v1
```

Comment:
These are in LFS. Do we want to change our rules so only screenshot tests go into LFS? Or are we okay sending all pngs?

Reply:
Hmm I don't have a strong opinion. I image the docs are going to either be viewed on github.com, which correctly renders the images inline, or on the published docs site, which will host the images. Is there a downside to storing these with LFS?

Reply:
Mostly just philosophical. It breaks the concept of a self-contained repo. The screenshot test goldens can be refreshed by running the tests. These are more like part of the codebase.

Reply:
@JakeWharton Good point, I'll clean up the code that generates these screenshots and add it to the repo.

Reply:
Good point, I'll clean up the code that generates these screenshots and add it to the repo.